development/tools/pkg/rendertemplates/genericcomponent.go

```diff
@@ -99,3 +50,61 @@ func (r *RenderConfig) GenerateComponentJobs(global map[string]interface{}) {
 		r.Values["JobConfigs"] = r.JobConfigs
 	}
 }
+
+// generateComponentJobs generates jobs for components
```

Comment:
What is global argument?

Reply:
It's a list of global values stored in `templates/config.yaml`